config.yaml

```diff
@@ -48,3 +48,9 @@ options:
     default: 1
     description: |
       Number of executors to configure for jenkins master.
+  public_url:
+    type: string
+    default: ""
+    description: >
```

Comment:
Please use a pipe here (for consistencies with other config keys in this file): 

"description: >" -> "description: |"